modules/custom/az_media/config/install/crop.type.az_extrasmall_full_width_bg.yml

```diff
@@ -0,0 +1,11 @@
+langcode: en
+status: true
+dependencies: {  }
+id: az_extrasmall_full_width_bg
```

Comment:
Minor nitpick: would be good to use consistent IDs (i.e. `bg` vs `background`)

Reply:
I think we might have a character limit infraction with background, so I could change them all to bg